### PR TITLE
Fix errors by linkchecker.py

### PR DESCRIPTION
**Pull request by atoato88**
This PR fixes errors which are shown by `scripts/linkchecker.py` on `en/docs/*`.

Fixed items:
- change to lower case in URL
- use relative paths

### CODE REVIEW

**Comment by atoato88** [comment]
/assign @zparnold
Could you review this PR?

File: content/en/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ each Pod in the scheduling queue according to constraints and available
 resources. The scheduler then ranks each valid Node and binds the Pod to a
 suitable Node. Multiple different schedulers may be used within a cluster;
 kube-scheduler is the reference implementation.
-See [scheduling](https://kubernetes.io/docs/concepts/scheduling-eviction/)
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/kube-scheduler.md:29]: This file is auto-generated, do fix it.

Comment (atoato88) [reply content/en/docs/reference/command-line-tools-reference/kube-scheduler.md:29]: Thank you for comment.
I'll fix this.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-27185--kubernetes-io-master-staging.netlify.app/docs/reference/using-api/deprecation-guide/

**Pull request merged**